### DISCUSSION TITLE
--global

### DISCUSSION THREAD

**Discussion by krisrajzlive** [parcel-bundler/parcel#9095] [Q&A]
I have an old app that has a script 

build: parcel build index.js --dist-dir ..//Scripts// -o ReactApp.Bundle.js --no-source-maps --detailed-report --log-level info --global appEntry

when I run the script, I get the error message unknown option --global. Has it been deprecated? If so, what is it's equivalent? I am using the latest version of parcel.

Thanks

**Comment by avoguga** [comment]
Hi there! U could see here in docs at migration:
https://parceljs.org/getting-started/migration/#--global

The --global flag has been removed. 

If u want to set the output format u have to set this with the target options.

https://parceljs.org/features/targets/#outputformat
